### ISSUE TITLE
Push Notification Button is not showing on iOS v12.0.1

### ISSUE THREAD

**Issue by nateblog** [OneSignal/react-native-onesignal#690]
**Description:**
I have recently tested latest iOS versions to check if the notification works perfectly on different iOS versions. It works on version 11.4 below; however, when I tested the notification version 12.0.1, there are **NO** action button showing up on the notification.

Is there some sort of code that I need to add to make the **notification action button** available on the latest iOS version? Please help

**Environment:**
macOS Mojave version 10.14.2
Xcode version 10.1
"react-native-onesignal": "^3.2.11",

**Steps to Reproduce Issue:**

Create a fresh react native app
Install react native onesignal
Run the react native in iOS v12.0.1
Send a push notification with actions buttons

**Comment by Nightsd01** [comment]
@nateblog It is possible that you are running into an [iOS 12 bug](https://github.com/OneSignal/OneSignal-iOS-SDK/issues/430) we discovered shortly after iOS 12.0.0 was first released.

Can you try adding an image to the push notification and tell me if the buttons do get updated afterwards?

Can you try setting the Deployment Target of the OneSignalNotificationServiceExtension (project settings in Xcode -> General tab) to iOS 10.0? If that does not work, change it to 11.0 and let me know if it works when you do this.

Can you try restarting the device to see if this makes any difference?

Even if one of those steps does _seem_ to fix the issue, it may just be fixed for a short time if it's the iOS 12 bug I mentioned above. To test this, if you get it fixed, completely uninstall and reinstall your app, register for notifications, and try the test again.

**Comment by nateblog** [comment]
@Nightsd01 - Thank you so so much for this helpful response . . Yes you are right, adding an image on the push notification seem to fix the issue temporarily.

After adding / attaching an image to the first notification, everything works fine on the second notification with or without adding / attaching an image.

I rebooted the device, reinstall the app and the issue showed up again.

Does this means we will wait for the apple guys to fix this bug? Thanks

**Comment by Nightsd01** [comment]
@nateblog unfortunately, yeah that's what it means. iOS 12 has had this issue since the first release - we've reported it to Apple but apparently someone had already reported it before we did. 

If it's not resolved soon I'll try to find other ways to reach out to them.

For what it's worth, I've only ever heard of this bug effecting apps loaded through Xcode & TestFlight, I've never heard of this bug effecting actual end users who install via the App Store.
